picard/cluster.py

```diff
@@ -318,6 +331,13 @@ def cluster(files, threshold):
 
             yield album_name, artist_name, (files[i] for i in album)
```

Comment:
`i` is already used in the same function, perhaps rename one of those to `j`, even though it has no impact on execution, it'd be more readable.

Reply:
OK